### ISSUE TITLE
thank you

### ISSUE THREAD

**Issue by iambumblehead** [jonsafari/mocp#5]
I understand this is not the official Music on Console repo, but since I have a github account already, I wanted to say a humble thank you. moc is one of my favourite applications.

**Comment by jonsafari** [comment]
Thanks for the thank you!  I feel the same way.  It would be nice to have the official MOC project on Github.  Current development of MOCP has stagnated in part by the use of SVN.  If you'd like to see MOC on github, please tell the lead developers at http://moc.daper.net/node/1520

**Comment by jonsafari** [comment]
The linked thread is almost two years old!  I was about to mention someone's libopusfile patch as another example, but then I realized that someone is you :smile: 

The fact that there are so many unofficial repos on Github is symptomatic of the current situation.  It would be much simpler to just fork an official Git project, make incremental changes, and put in a pull request.  I would love to contribute more, but I'm not going to touch SVN :relaxed: 

There will always be ongoing issues, and as more people use MOC, the more issues there will be.  The easier it is to branch/fix/merge often, the quicker development will be in the long run.

I've created a org account [music-on-console](https://github.com/music-on-console), and would be happy to add you, jcf, and any other contributors.

**Comment by tomaszg7** [comment]
Yeah, I can only quote jcf from the thread: "I agree with everything you say" ;) Even if for a some time MOC development would need to happen in SVN, that doesn't preclude the existence of official MOC repo, which would mirror it.

Feel free to pull opus decoder from my repo to yours, if you wish.

**Comment by iambumblehead** [comment]
There are lurkers like myself who might come out to contribute something if moc were on gitlab or github. 

For example [mnemosyne](https://github.com/mnemosyne-proj/mnemosyne/issues) was developed away from github, but appeared on github recently. When I saw it, I [tried to build a snap package](https://github.com/mnemosyne-proj/mnemosyne/issues/37) for it. Snap packaging still fails on the newest beta Ubuntu Bionic Beaver, but my point is that I would not have tried if I hadn't seen mnemosyne on github.

**Comment by tomaszg7** [comment]
Yeah, on top of it all, engine for forum on MOC's webpage is really weak... For example, there are no notifications.

**Comment by happydig** [comment]
Thanks!
